api/types.proto

```diff
@@ -329,6 +329,8 @@ message RestartPolicy {
 
 	// Delay between restart attempts
 	// Note: can't use stdduration because this field needs to be nullable.
+	// Note: If backoff is non null then Delay will be added to the calculated
```

Comment:
That is a bit surprising. It would be better to have one supplant the other.

Reply:
Delay is setup to be a base offset on top of the expo-backoff in response to an earlier comment by @dperny.  Originally we had a separate base offset which was a part of the backoff policy, and would override delay.